benchmarks/graphics_pipeline/GraphicsBenchmarkApp.h

```diff
@@ -0,0 +1,242 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef BENCHMARKS_GRAPHICS_PIPELINE_GRAPHICS_BENCHMARK_APP_H
+#define BENCHMARKS_GRAPHICS_PIPELINE_GRAPHICS_BENCHMARK_APP_H
+
+#include "FreeCamera.h"
+#include "MultiDimensionalIndexer.h"
+
+#include "ppx/grfx/grfx_config.h"
+#include "ppx/knob.h"
+#include "ppx/math_config.h"
+#include "ppx/ppx.h"
+
+#include <array>
+#include <vector>
+
+#if defined(USE_DX12)
+const grfx::Api kApi = grfx::API_DX_12_0;
+#elif defined(USE_VK)
+const grfx::Api kApi = grfx::API_VK_1_1;
+#endif
+
+static constexpr uint32_t kMaxSphereInstanceCount  = 3000;
+static constexpr uint32_t kSeed                    = 89977;
+static constexpr uint32_t kMaxFullscreenQuadsCount = 1000;
+
+static constexpr char* kShaderBaseDir = "benchmarks/shaders";
```

Comment:
this produces a compiler warning. You need to add a const after constexpr
https://stackoverflow.com/questions/54258241/warning-iso-c-forbids-converting-a-string-constant-to-char-for-a-static-c

Reply:
Thanks, I'll change this. Did you see this warning on Linux? I don't think I am able to see this on Windows when building with VS

Reply:
Yes, on linux